lncrawl/core/crawler.py

```diff
@@ -115,6 +118,21 @@ def download_chapter_body(self, chapter) -> str:
         pass
     # end def
 
+    def download_image(self, url) -> bytes:
+        assert isinstance(url, str)
+        if len(url) > 1000 or url.startswith('data:'):
+            content = base64.b64decode(url.split('base64,')[-1])
+        else:
+            logger.info('Downloading image: ' + url)
+            response = self.crawler.get_response(url, headers={
```

Comment:
this should be self.get_response

Reply:
> But to fix some PNG image not working I am modify saving image statement to force conversion image to RGB mode.

Now it looks good to me. There is a slight issue I mentioned in the comment above